extensions/amp-sidebar/0.1/test/test-amp-sidebar.js

```diff
@@ -618,13 +636,23 @@ describes.realWin(
           });
           owners.scheduleLayout = sandbox.stub();
 
-          impl.open_();
+          execute(impl, 'open', /* trust */ 123);
```

Comment:
Nit: Can we just use the `ActionTrust.HIGH` and `ActionTrust.LOW` enums here to make this more readable? I think they're already imported at the top of this file. =)

Reply:
I intentionally used a bogus value to make sure that the trust is being propagated and not hard-coded. :)